services/wallet/activity/activity.go

```diff
@@ -623,6 +624,10 @@ func getActivityEntries(ctx context.Context, deps FilterDependencies, addresses
 		getActivityType := func(trType sql.NullByte) (activityType Type, filteredAddress eth.Address) {
 			if trType.Valid {
 				if trType.Byte == fromTrType {
+					match, _ := regexp.MatchString("0x0+$", toAddress.String())
```

Comment:
Hmm a contract deployment activity should be:
- A `Send` activity
- That has `type` == `eth`
- And `contract_address` != null address
The toAddress == null address condition could be a token burn, for example